include/drawtypes/label.hpp

```diff
@@ -18,6 +18,11 @@ namespace drawtypes {
     string suffix{""};
   };
 
+  struct sides {
+    int left;
+    int right;
+  };
+
```

Comment:
you could use the type in `components/types.hpp`:

~~~cpp
struct side_values;
~~~